packages/eslint-plugin-office-addins/src/rules/call-sync-before-read.ts

```diff
@@ -30,6 +34,8 @@ export = {
       switch (topExpression.parent?.type) {
         case TSESTree.AST_NODE_TYPES.AssignmentExpression:
           return topExpression.parent.right === topExpression;
+        case TSESTree.AST_NODE_TYPES.CallExpression:
+          return isProperty(node);
```

Comment:
The method is for checking "isRead" . . . now we are checking something else.  It doesn't belong here.

Reply:
With the other changes, did this get solved? I didn't get this comment very much

Reply:
Nope . . .

The switch statement is in a method call 'checkPropertyIsRead' and you are adding a case that has nothing to do with weather a property is being read or written to.

Reply:
It refers to the case that in fact, it is a property that is there.
The method needs to check if the "thing" being read is a property or not and also check if it is being read after confirming it is a property. I still like the way it is, do you think it deserves some function name change?

Reply:
Took this offline to get a clear picture of our different points of view.